src/api/app/models/bs_request_action.rb

```diff
@@ -822,6 +822,25 @@ def uniq_key
     ' '
   end
 
+  def name
+    uniq_key
+  end
+
+  def details
```

Comment:
All of this sounds like view code to me...

Reply:
Part of this code is now moved to the view